### PR TITLE
docs: add scope to readme

### PR DESCRIPTION
**Pull request by pevisscher**

### CODE REVIEW

**Comment by pevisscher** [comment]
Current dependencies on/for this PR:
* main
  * **PR #59** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/59" target="_blank">Graphite</a>  👈
    * **PR #60** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/60" target="_blank">Graphite</a> 
      * **PR #61** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/61" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/event-horizon/59?utm_source=stack-comment).

**Review by loredanamoanga — APPROVED** [review]

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/event-horizon/59?utm_source=stack-comment).


Job Reference: oPCx7VL6iilScTtJLPDq

**Pull request merged**
